<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWork</class>
 <widget class="QDialog" name="EditWork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="statusBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
